pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[tool.ruff]
+lint.select = ["PL", "YTT", "BLE", "I", "UP", "A", "COM", "T10", "EXE", "TID", "F401", "F402", "F821"]
+
+lint.ignore = ["E501", "PLR2004", "COM812", ]
+
+target-version = "py310"
+
+extend-exclude = [
+    "docker/alternator-dns",
+]
+respect-gitignore = true
+force-exclude = true
+line-length = 112
```

Comment:
@vponomaryov 

look at the end results, on the commit that does the formatting, before it's not really used by the linter

Reply:
Confusing to see 2 different definitions in scope of single PR, but ok.